packages/reports/src/Components/new-affiliate-account-modal.jsx

```diff
@@ -0,0 +1,58 @@
+import React from 'react';
+import { Icon, Modal, Text, Button } from '@deriv/components';
+import { localize, Localize } from '@deriv/translations';
+import { connect } from 'Stores/connect';
+import PropTypes from 'prop-types';
+import '../sass/app/_common/components/new-affiliate-account-modal.scss';
```

Comment:
Do we need to report this stylesheet in the component? Seems like other components in reports package have styles too but there is no import statement required in the `jsx` file

Reply:
yes, we need this, if I don't import that it wouldn't work